src/main/java/file/notes/App.java

```diff
@@ -18,10 +18,15 @@ public static void main(String[] args) {
             } catch (IOException e) {
                 e.printStackTrace();
             }
-            System.out.println("Success: " + command);
             if ("q".equals(command)) {
                 System.exit(0);
             }
+            else if ("create".equals(command) || "delete".equals(command) || "show".equals(command)) {
+                System.out.println("Success: " + command);
+            }
+            else {
+                System.out.println("Error. Unsupported command. Press one of commands: creade, delete, show." );
```

Comment:
You can have a constant (https://www.java-tips.org/java-se-tips-100019/24-java-lang/802-how-do-i-declare-a-constant-in-java.html) List that contains all the commands and just print this list. This way you won't make a typo like "creade".